cocos/network/WebSocket.h

```diff
@@ -238,7 +239,7 @@ class CC_DLL WebSocket
 
     struct lws*         _wsInstance;
     struct lws_context* _wsContext;
-    std::shared_ptr<bool> _isDestroyed;
+    std::shared_ptr<std::atomic<bool>> _isDestroyed;
```

Comment:
Since there isn't more than twice to set the _isDestroyed, to make `std::shared_ptr<bool>` to `std::shared_ptr<std::atomic<bool>>` isn't needed yep.

But for best practice, it's better to make it as `atomic` since the variable is used in different thread. :)